system_test.go

```diff
@@ -241,20 +241,27 @@ func TestHost(t *testing.T) {
 	info := host.Info()
 	assert.NotZero(t, info)
 
-	loadAvg, err := host.LoadAverage()
-	if errors.Is(err, types.ErrNotImplemented) {
-		t.Log("Load average not implemented")
-		return
-	}
+	output := map[string]interface{}{}
+	output["host.info"] = info
 
-	if err != nil {
-		t.Fatal(err)
+	if v, ok := host.(types.LoadAverage); ok {
+		loadAvg, err := v.LoadAverage()
+		if errors.Is(err, types.ErrNotImplemented) {
```

Comment:
This is not needed given that not of the implementations return this error.

Reply:
You're right, didn't think about it, my bad!

Everything should be ok now.